README.md

```diff
@@ -406,8 +406,10 @@ geometries in Well Known Text (WKT) and GeoJSON format:
   interactions. See example below:
 
```

Comment:
Update CHANGELOG.md to communicate explicitly that 2.x has breaking changes!

Reply:
I've force-pushed a version of this change that includes that notice;

```diff
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 8f53089..2d375d9 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Lazily load behaviors as separate Webpack chunks.
 - Decouple map instance creation/management from `window.farmOS.map`.
 - Distribute CSS as separate files/chunks instead of bundled with the JS.
+- **BREAKING CHANGES:** Note: As one might expect from a new major version, there are breaking changes involved in moving from 1.x to 2.x of farmOS-map. See the [README for upgrade instructions](./README.md#upgrading-from-farmos-map-1x-to-2x).
 
 ### Fixed
 
```